game/game.go

```diff
@@ -375,6 +396,54 @@ type Game struct {
 	FinishedAgo time.Duration `datastore:"-" ticker:"true"`
 }
 
+func (g *Game) canMergeInto(o *Game, avoid *auth.User) bool {
+	if g.NoMerge || o.NoMerge {
+		return false
+	}
+	if g.Started || o.Started {
+		return false
+	}
+	if g.Closed || o.Closed {
+		return false
+	}
+	if g.Finished || o.Finished {
+		return false
+	}
+	if g.Variant != o.Variant {
+		return false
+	}
+	if g.PhaseLengthMinutes != o.PhaseLengthMinutes {
+		return false
+	}
+	if g.MaxHated != o.MaxHated {
+		return false
+	}
+	if g.MaxHater != o.MaxHater {
+		return false
+	}
+	if g.MinRating != o.MinRating {
+		return false
+	}
+	if g.MaxRating != o.MaxRating {
+		return false
+	}
+	if g.MinReliability != o.MinReliability {
+		return false
+	}
+	if g.MinQuickness != o.MinQuickness {
```

Comment:
You could golf this a bit by using a few or statements.

Reply:
Yeah I know, but this felt slightly more obvious - now you can quickly visually inspect that each side of the != is the same field.